modules/local/subworkflow/phylogenies.nf

```diff
@@ -0,0 +1,32 @@
+/*
+ * Phylogenies subworkflow
+ */
+
+params.fasttree_options    = [:]
+//params.iqtree_options    = [:]
+
+include { FASTTREE } from '../fasttree' addParams( options: params.fasttree_options )
+//include { IQTREE  } from  '../iqtree' addParams( options: params.iqtree_options )
+
+
+workflow PHYLOGENIES {
+    take:
+    fasta     // channel: aligned pseudogenomes or filtered version
+    
+    main:
+    /*
+     * MODULE Fasttree
+     */
+    FASTTREE ( fasta )
```

Comment:
What do you think about each TREE_ALGORITHM_options have a key of `build` which is true or false by default?
```suggestion
if (params.fasttree_options.build){
        FASTTREE ( fasta )
}
```